study-builder/studies/basil/i18n/en.conf

```diff
@@ -256,242 +256,255 @@
-    "nsclc__lung_large_cell_carcinoma": "NSCLC - lung large cell carcinoma",
-    "nsclc__lung_adenocarcinoma": "NSCLC - lung adenocarcinoma",
-    "additional_cancers__other_cancer_types__rare_cancers": "Additional Cancers / Other Cancer Types / Rare Cancers",
-    "desmoidtype_fibrosis_df": "Desmoid-type Fibrosis (DF)",
-    "giant_cell_tumor_of_the_bone_gct": "Giant cell tumor of the bone (GCT)",
-    "paraganglioma": "Paraganglioma",
-    "phenochromocytoma": "Phenochromocytoma",
-    "primary_peritoneal_cancer_ppc": "Primary Peritoneal cancer (PPC)",
-    "epithelioid_hemangioendothelioma_ehe": "Epithelioid hemangioendothelioma (EHE)",
-    "desmoplastic_small_round_cell_tumor_dsrct": "Desmoplastic small round cell tumor (DSRCT)",
-    "cancer_of_unknown_primary_cup": "Cancer of Unknown Primary (CUP)",
-    "adenoid_cystic_carcinoma_acc": "Adenoid cystic carcinoma (ACC)",
-    "ameloblastic_carcinoma": "Ameloblastic carcinoma",
-    "apocrine_carcinoma": "Apocrine carcinoma ",
-    "brca1_hereditary_breast_and_ovarian_cancer_syndrome_brca1_hboc": "BRCA1 hereditary breast and ovarian cancer syndrome (BRCA1 HBOC)",
-    "brca2_hereditary_breast_and_ovarian_cancer_syndrome_brca2_hboc": "BRCA2 hereditary breast and ovarian cancer syndrome (BRCA2 HBOC)",
-    "carcinoid_syndrome": "Carcinoid Syndrome",
-    "collecting_duct_carcinoma_cdc": "Collecting duct carcinoma  (CDC)",
-    "dermal_eccrine_cylindroma": "Dermal eccrine cylindroma",
-    "dermatofibrosarcoma_protuberans": "Dermatofibrosarcoma protuberans",
-    "dicer1related_pleuropulmonary_blastoma_cancer_predisposition_syndrome": "DICER1-related pleuropulmonary blastoma cancer predisposition syndrome ",
-    "eccrine_porocarcinoma": "Eccrine porocarcinoma",
-    "fibrolamellar_carcinoma_flc": "Fibrolamellar Carcinoma (FLC)",
-    "goblet_cell_carcinoid_gcc": "Goblet cell carcinoid (GCC)",
-    "hidradenocarcinoma": "Hidradenocarcinoma",
-    "mucoepidermoid_carcinoma": "Mucoepidermoid carcinoma ",
-    "myoepithelial_carcinoma": "Myoepithelial carcinoma ",
-    "pineoblastoma": "Pineoblastoma",
-    "pituitary_cancer": "Pituitary cancer",
-    "primary_bone_cancer": "Primary bone cancer ",
-    "sertolileydig_cell_tumors": "Sertoli-leydig cell tumors ",
-    "sinonasal_undifferentiated_carcinoma_snuc": "Sinonasal undifferentiated carcinoma (SNUC)",
-    "stewart_treves_syndrome": "Stewart Treves syndrome ",
-    "thyroid_cancer_carcinoma": "Thyroid cancer (carcinoma) ",
-    "von_hippellindau_disease_vhl": "Von hippel-lindau disease (VHL)",
-    "other_cancer": "Other cancer",
-    "other_cancer_details": "Other cancer details",
+    "cancer_group__brain__brain_or_cns_tumors": "Brain or Central Nervous System tumors",
+    "cancer__brain__atrt__atypical_teratoid_tumor": "Atypical teratoid rhabdoid tumor (ATRT)",
+    "cancer__brain__anapl_astrocytoma": "Anaplastic astrocytoma",
+    "cancer__brain__astrocytoma_other": "Astrocytoma, other / Cerebellar astrocytoma / Cerebral astrocytoma",
```

Comment:
Let's make "Cerbellar astrocytoma" and "Cerebral astrocytoma" separate and indvidual options.
Looks like we are missing the standalone "Astrocytoma".
Keep "Astrocytoma, other" (I will see if they are willing to remove from list but keep it for now)

Reply:
Updated the cancer list